scss/titles.scss

```diff
@@ -0,0 +1,5 @@
+.title {
+  font-family: "Roboto", sans-serif;
+  font-size: 24px;
+  margin: 5% 7%;
```

Comment:
we'll add the margin later, the idea here is to create a title component that can be replicated everywhere. Margins are related to the context, this means that depending on where we place the component the margin we'll wat will be different. So until we add the context later on, we should avoid adding margins.